<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabOCTA">
       <attribute name="title">
        <string>OCTA</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <layout class="QGridLayout" name="gridOCTA">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item row="4" column="2">
           <widget class="QGroupBox" name="groupControl">
            <property name="title">
             <string>Control</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Scan</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Acquire</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="2">
           <widget class="QGroupBox" name="groupConfig">
            <property name="title">
             <string>Configuration</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLeft">
               <item row="0" column="0">
                <widget class="QComboBox" name="comboObjective"/>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="labelObjective">
                 <property name="text">
                  <string>Objective</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QComboBox" name="comboRate"/>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="labelRate">
                 <property name="text">
                  <string>A-line rate</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QComboBox" name="comboApodization"/>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="labelApodization">
                 <property name="text">
                  <string>Apodization window</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QFormLayout" name="formRight">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridProc">
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="checkInterpolation">
                   <property name="text">
                    <string>Lambda-k interpolation</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="checkFFT">
                   <property name="text">
                    <string>Fourier transform</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QCheckBox" name="checkBox">
                   <property name="text">
                    <string>32 bit (default 64)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="labelProcessing">
                 <property name="text">
                  <string>Processing</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1" rowspan="2">
           <widget class="QGroupBox" name="groupRepeats">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>A-line repeats</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="2" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>Averaging</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QSpinBox" name="spinBox_2"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" rowspan="2">
           <widget class="QGroupBox" name="groupScan">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Scan geometry</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="1">
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLineEdit" name="lineEdit"/>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Horizontal A-line spacing</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLineEdit" name="lineEdit_3"/>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Vertical A-line spacing</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout_7">
               <item row="0" column="0">
                <widget class="QSpinBox" name="spinBox_3"/>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>ROI width</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="spinBox_4"/>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string># pixels width</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QSpinBox" name="spinBox_5"/>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>ROI height</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QSpinBox" name="spinBox_6"/>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string># pixels height</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupExperiment">
            <property name="title">
             <string>File</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formExp">
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridExp">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="lineExp"/>
                 </item>
                 <item row="0" column="1">
                  <widget class="QToolButton" name="buttonBrowse">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="labelExp">
                 <property name="text">
                  <string>Experiment directory</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLineEdit" name="lineFileName"/>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="labelFileName">
                 <property name="text">
                  <string>Trial name</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QComboBox" name="comboFileType"/>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="labelFileType">
                 <property name="text">
                  <string>File type</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QSpinBox" name="spinBox_7"/>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Maximum file size</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFig8A">
       <attribute name="title">
        <string>Fig8A</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCircleA">
       <attribute name="title">
        <string>CircleA</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFile"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPreferences"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSpectralRadar">
   <property name="text">
    <string>SpectralRadar</string>
   </property>
  </action>
  <action name="actionDefaults">
   <property name="text">
    <string>Defaults</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
